website/addons/dataverse/static/dataverseFangornConfig.js

```diff
@@ -124,7 +124,7 @@ var _dataverseItemButtons = {
                 }, options)
             );
         }
-        if (item.kind === 'folder' && item.data.addonFullname && item.data.version === 'latest' && item.data.permissions.edit) {
+        if (item.kind === 'folder' && item.data.addonFullname && item.data.version === 'latest' && item.data.permissions.edit && tb.options.placement !== 'fileview') {
```

Comment:
This condition is getting fairly long - since you've added the `tb.options.placement !== 'fileview'` to all three branches of this structure, perhaps wrapping the entire structure in a second conditional would be more clear?

```javascript
if (tb.options.placement !== 'fileview') {
    // Code prior to this change
}
```

Also, while it's not sufficient justification for changing things in and of itself, your current structure has three comparisons that the interpreter will have to make for `tb.options.placement !== 'fileview'`. By moving it outside, it'll only be made once - and if it fails, the entire block will be skipped.